pepper-apis/src/main/java/org/broadinstitute/ddp/filter/OnlyStudyAdminFilter.java

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.ddp.filter;
+
+import static spark.Spark.halt;
+
+import org.broadinstitute.ddp.constants.RouteConstants.PathParam;
+import org.broadinstitute.ddp.security.DDPAuth;
+import org.broadinstitute.ddp.util.RouteUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Filter;
+import spark.Request;
+import spark.Response;
+
+/**
+ * A filter that only allows access to study admins.
+ */
+public class OnlyStudyAdminFilter implements Filter {
+
+    private static final Logger LOG = LoggerFactory.getLogger(OnlyStudyAdminFilter.class);
+    private static final AuthPathRegexUtil pathMatcher = new AuthPathRegexUtil();
+
+    @Override
+    public void handle(Request request, Response response) {
+        DDPAuth ddpAuth = RouteUtil.getDDPAuth(request);
```

Comment:
Have we established that our existing decoding of Auth0 tokens will work unchanged for BWH study admins? I believe were talking about authenticating CRCs against a different system. This is a question. I still don't know how this is supposed to work.

Reply:
Not sure. I will update as we learn more about that.